artemis/modules/data/wappalyzer/main.go

```diff
@@ -0,0 +1,79 @@
+package main
```

Comment:
hmm, I don't think this belongs to modules/data as it's code, maybe modules/utils?